modules/dev-tools/templates/.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+const typescriptConfigs = require('@typescript-eslint/eslint-plugin').configs;
+
+module.exports = {
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018
+  },
+  plugins: ['markdown'],
+  extends: [
+    'uber-es2015',
+    'prettier'
+  ],
+  rules: {
+    'guard-for-in': 0,
+    'no-inline-comments': 0,
+    'camelcase': 0
+  },
+  overrides: [
+    {
+      files: ['**/*.ts', '**/*.tsx'],
```

Comment:
Some comments on these overrides, why we do need these would be really helpful. You have done a lot of research here, you could help future maintainers by explaining e.g why we need `sourceType: module`

Reply:
added a few